packages/iov-core/src/multichainsigner.spec.ts

```diff
@@ -142,6 +144,13 @@ describe("MultiChainSigner", () => {
       expect(acct).toBeTruthy();
       expect(acct.data.length).toBe(1);
       expect(acct.data[0].balance.length).toBe(1);
+
+      const faucetAddr2 = signer.keyToAddress(ethId, faucet.pubkey);
```

Comment:
This cannot work because the Ethereum codec cannot calculate an address from an Ed25519 pubkey (thus CI fail). We could use

```ts
const ganacheMainIdentity: PublicIdentity = {
  pubkey: {
    algo. Algorithm.Secp256k1,
    data: Encoding.fromHex("04965fb72aad79318cd8c8c975cf18fa8bcac0c091605d10e89cd5a9f7cff564b0cb0459a7c22903119f7a42947c32c1cc6a434a86f0e26aad00ca2b2aff6ba381") as PublicKeyBytes,
  }
}
```

instead of `faucet`